benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java

```diff
@@ -249,7 +249,7 @@ private List<QueryExecutionResult> executeConcurrentQueries(Benchmark benchmark,
                         firstQuery = false;
                     }
                     try {
-                        queryExecutionResults.add(executeSingleQuery(queryExecution, benchmark, connection, false, executionTimeLimit));
+                        queryExecutionResults.add(executeSingleQuery(queryExecution, benchmark, connection, warmup, executionTimeLimit));
```

Comment:
Why this change is needed since you added `&& !benchmark.isThroughputTest()`?

Wouldn't you get same result by leaving `false` here?

Reply:
If you set `warmup` to `false` you have at least two bugs in two different code paths:
1. Code is trying to report that **benchmark execution** was started despited of the fact **benchmark** (it is a higher level term) is not started. It is a one problem.
2. The second one is when we try to report `executionStarted` twice. 
 
If we set the `warmup` to the `true` it also resolved the issue. But, it is not a good idea to lie because, as you see, it is very errorneous to make things implicitly instead of explicitly like here: 
"Do not report that stuff for `throughputTest`" instead of trying to use another parameter in specific way.